ga4gh/protocol.py

```diff
@@ -23,6 +24,63 @@ def convertDatetime(t):
     return int(millis)
 
 
+class SearchResponseBuffer(object):
+    """
+    A buffer to hold the string representation of protocol Response
+    objects. This allows us to sequentially build responses, and
+    reduces the number of live objects required to build response
+    objects.
+    """
+    def __init__(self, pageListName, pageSize, maxResponseLength=None):
+        self._pageListName = pageListName
+        self._pageSize = pageSize
+        self._maxResponseLength = 2**32
```

Comment:
@dcolligan  I don't see a `maxResponseLength` constant (or anything similar) in `backend.py`... which one do you mean?